python/cairo-core/tests/src/cairo_core/test_maths.py

```diff
@@ -182,11 +182,16 @@ def test_felt252_to_bytes_be_should_panic_on_wrong_output_noncanonical(
         ), cairo_error(message="felt252_to_bytes_be: byte not in bounds"):
             cairo_run("test__felt252_to_bytes_be", value=value, len=len_)
 
-    @given(value=st.integers(min_value=0, max_value=DEFAULT_PRIME - 1))
+    @given(value=felt)
     def test_felt252_bit_length(self, cairo_run, value):
-        res = cairo_run("felt252_bit_length", value=value)
+        res = cairo_run("felt252_bit_length", value)
         assert res == value.bit_length()
 
+    @given(value=felt)
+    def test_felt252_bytes_length(self, cairo_run, value):
```

Comment:
can you add an `@example(value=PRIME)` to ensure coverage of the bytes_length == 32 case?

Reply:
or anything that would trigger proper coverage